src/parsing.jl

```diff
@@ -59,19 +59,27 @@ function replace_dotted!(membernames, e)
     Expr(:., x_new, y_new)
 end
 
+composed_or_symbol(x) = false
+composed_or_symbol(x::Symbol) = true
+function composed_or_symbol(x::Expr)
+    x.head == :call &&
+        x.args[1] == :∘ &&
+        all(composed_or_symbol, x.args[2:end])
```

Comment:
`[2:end]` selector is not needed as `:∘` is a symbol. Bit can stay as it is also.